float64 gpsWeek
float64 secInWeek
float32 fracOfSec
float64 rosTime
float32 estimatedTW